_infra/helm/collection-exercise/templates/loadbalancer.yml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: collection-exercise-lb
+  annotations:
+    networking.gke.io/load-balancer-type: "Internal"
+    networking.gke.io/load-balancer-ip-addresses: {{ .Values.loadBalancer.ipAddress }}
```

Comment:
Worth noting that this is favoured over `spec.loadBalancerIP` for GKE > 1.2X(?). They're not both required but we will be running different versions of (GKE some < 1.2X ?) so until we move completely off the deprecated version I guess we can have both settings in place (even if it looks confusing)